composer.json

```diff
@@ -67,6 +67,7 @@
     },
     "conflict": {
         "doctrine/doctrine-bundle": ">=3",
+        "psr/container": "1.1.0 || 2.0.0",
```

Comment:
```suggestion
        "psr/container": "1.1.0 || >=2.0.0",
```

Reply:
Even if we haven't a clear idea of how this will be resolved, I think it's a good idea to widening the 2.x version by now.